jbmc/src/java_bytecode/ci_lazy_methods_needed.cpp

```diff
@@ -42,6 +61,14 @@ bool ci_lazy_methods_neededt::add_needed_class(
   if(symbol_table.symbols.count(cprover_validate))
     add_needed_method(cprover_validate);
 
+  // Special case for enums. We may want to generalise this, see TG-4689
```

Comment:
Generalise in what way? Since this is in JBMC, the TG reference is not very helpful.

Reply:
There is a link to `java_object_factoryt::gen_nondet_pointer_init`, which has more details. I slightly rephrased this comment, putting the link first and the ticket number only afterwards in brackets so it's clearer that you don't actually need access to the ticket to work on this.